src/tools/dartpad/troubleshoot.md

```diff
@@ -24,7 +24,7 @@ The dart.dev homepage and many codelabs have embedded DartPads.
 If these DartPads don't appear at all,
 then try the following:
 
-* If you use an ad blocker, try disabling it.
+* Disable your ad blocker if you are using one.
```

Comment:
Our style is to put the "if" first:  https://developers.google.com/style/clause-order

If you're using an ad blocker, disable it.